tests/e2e/e2e.test.ts

```diff
@@ -155,6 +156,7 @@ it('When bundled custom scalars are used the project type checks and generates e
             data: {
               id: 'foo1',
               someDateTimeField: new Date("2021-05-10T20:42:46.609Z"),
+              someDecimalField: new Prisma.Decimal(24.454545),
```

Comment:
Create this decimal value using the `decimal.js` dependency instead of Prisma's bundled version.